soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
@justbuchanan Having looked over the code again, yes the c++ side still uses -1 which is intentional. The issue is that I forgot to go through the python side to change all of the checks to use -1 so I will do that now.